translations/messages.fr.xlf

```diff
@@ -2287,6 +2295,82 @@
+        <target>Lasca est un jeu basé sur le jeu de dames, crée en 1911 par Emanuel Lasker, champion du monde d'échecs. Lasca se joue sur un plateau de 7x7 cases, chaque joueur a 11 pièces.</target>
+      </trans-unit>
+      <trans-unit id="8228991594058357867" datatype="html">
+        <source>The goal of Lasca is, like for draughts, to render the opponent out of possible move, either by capturing all his pieces, either by blocking them.</source>
+        <target>Le but de Lasca est, comme aux dames, de rendre l'adversaire incapable de jouer, soit en capturant toutes ses pièces, soit en les bloquant</target>
+      </trans-unit>
+      <trans-unit id="1534367694207535197" datatype="html">
+        <source>Steps</source>
+        <target>Pas</target>
+      </trans-unit>
+      <trans-unit id="4701749826555333352" datatype="html">
+        <source>A simple step is made by one diagonal move forward, left or right. Click on the chosen piece, then on it&apos;s landing square.&lt;br/&gt;&lt;br/&gt;You are playing Dark, do the first move.</source>
+        <target>Un simple pas est un déplacement de un en diagonal, en avant, vers la gauche ou la droite, cliquez sur une pièce au choix, puis sur sa case d&apos;arrivée.&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faites le premier mouvement.</target>
+      </trans-unit>
+      <trans-unit id="7763413917819120355" datatype="html">
+        <source>You have to capture when you can. It is the case here, so click on the piece that must capture, and then on its landing square.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, go ahead.</source>
+        <target>Vous devez capturer quand vous le pouvez, c'est le cas ici, cliquez sur la pièce qui doit capturer, et ensuite sur sa case d'atterrissage&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, allez-y.</target>
+      </trans-unit>
+      <trans-unit id="3662184238859847370" datatype="html">
+        <source>Congratulations, notice that the captured piece was not removed from the board, but put below the capturing pieces.</source>
+        <target>Félicitations, constatez que la pièce capturée n'as pas été enlevée du plateau, mais placée en dessous de la pièce qui l'as capturé.</target>
+      </trans-unit>
+      <trans-unit id="1430493612123692909" datatype="html">
+        <source>Multiple captures (1/2)</source>
+        <target>Captures multiples (1/2)</target>
+      </trans-unit>
+      <trans-unit id="6157883150195310948" datatype="html">
+        <source>If, after the beginning of your capture, the piece that just moved can capture another piece, it has to capture until she can no longer. To do so, you must then click again on the next landing space. Note that, when capturing with an officer, you cannot jump twice over the same stack.&lt;br/&gt;&lt;br/&gt;You are playing Dark, a double capture is possible, go ahead.</source>
```

Comment:
she -> it, AGAIN
"until it can no longer" manque une fin ici

Reply:
ça doit capturer jusqu'à ne plus pouvoir.
ça doit capturer jusqu'à ne plus pouvoir capturer.

C'est souvent formulé comme ça aussi en anglais, ça marche pas ? Après certes, être explicite est bien?

Reply:
ouais mieux vaut expliciter, et ça me sonne louche même si je sais pas trop si c'est incorrect, mais TPR du coup